2025-09-30T04:48:23 {"cpu": 12.82568359375, "cpu_total": 8, "cpu_usage": 1.60321044921875, "memory": 6840.0, "memory_total": 11944.0, "memory_usage": 0.5726724715338245, "pipeline_run_id": 393, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "2b86a123826c", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 393 in schedule 5 is alive.", "timestamp": 1759207703.581858, "uuid": "5c6705663aa44b508c56a9ffd7616dd2"}
2025-09-30T04:48:25 {"block_run_id": 1124, "block_uuid": "ingest_taxi_trips", "pipeline_run_id": 393, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "2b86a123826c", "level": "INFO", "message": "Execute PipelineRun 393, BlockRun 1124: pipeline ny_taxi_pipeline block ingest_taxi_trips", "timestamp": 1759207705.110758, "uuid": "035a428a9fbb4d99b549dba027b349ff"}
2025-09-30T04:48:25 {"cpu": 12.82568359375, "cpu_total": 8, "cpu_usage": 1.60321044921875, "memory": 6905.0, "memory_total": 11944.0, "memory_usage": 0.5781145344943067, "pipeline_run_id": 393, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "2b86a123826c", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 393 in schedule 5 is alive.", "timestamp": 1759207705.237574, "uuid": "7e0995ef95f740988fdef0a56858a6d5"}
2025-09-30T04:48:33 {"cpu": 12.751953125, "cpu_total": 8, "cpu_usage": 1.593994140625, "memory": 7018.0, "memory_total": 11944.0, "memory_usage": 0.5875753516409913, "pipeline_run_id": 393, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "2b86a123826c", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 393 in schedule 5 is alive.", "timestamp": 1759207713.957263, "uuid": "4b70853b10bf4646bb35d53fd4fec039"}
2025-09-30T12:35:04 {"cpu": 0.20654296875, "cpu_total": 8, "cpu_usage": 0.02581787109375, "memory": 1199.0, "memory_total": 11944.0, "memory_usage": 0.10038513060951106, "pipeline_run_id": 393, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 393 in schedule 5 is alive.", "timestamp": 1759235704.49812, "uuid": "b1b28748c97b41ab9e3ab2e9f63eb9ca"}
2025-09-30T12:35:05 {"block_run_id": 1124, "block_uuid": "ingest_taxi_trips", "pipeline_run_id": 393, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Execute PipelineRun 393, BlockRun 1124: pipeline ny_taxi_pipeline block ingest_taxi_trips", "timestamp": 1759235705.046217, "uuid": "b3dc967799c84beab39887a096eafecc"}
2025-09-30T12:35:14 {"cpu": 0.2744140625, "cpu_total": 8, "cpu_usage": 0.0343017578125, "memory": 1471.0, "memory_total": 11944.0, "memory_usage": 0.12315807099799063, "pipeline_run_id": 393, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 393 in schedule 5 is alive.", "timestamp": 1759235714.05377, "uuid": "e8efe76858264f6bb7ccb2448a185b6c"}
2025-09-30T12:35:23 {"cpu": 0.34130859375, "cpu_total": 8, "cpu_usage": 0.04266357421875, "memory": 1554.0, "memory_total": 11944.0, "memory_usage": 0.13010716677829873, "pipeline_run_id": 393, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 393 in schedule 5 is alive.", "timestamp": 1759235723.969817, "uuid": "e23a642cfce8483d9f15f7906ec4a0a9"}
2025-09-30T12:35:34 {"cpu": 0.35986328125, "cpu_total": 8, "cpu_usage": 0.04498291015625, "memory": 1645.0, "memory_total": 11944.0, "memory_usage": 0.13772605492297388, "pipeline_run_id": 393, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 393 in schedule 5 is alive.", "timestamp": 1759235734.102929, "uuid": "d089fefd327e4cb785491a36c9aeb904"}
2025-09-30T12:35:44 {"cpu": 0.431640625, "cpu_total": 8, "cpu_usage": 0.053955078125, "memory": 1720.0, "memory_total": 11944.0, "memory_usage": 0.14400535833891492, "pipeline_run_id": 393, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 393 in schedule 5 is alive.", "timestamp": 1759235744.168612, "uuid": "f3f8fd4828dc442da1464c68258320c9"}
2025-09-30T12:35:54 {"cpu": 0.48095703125, "cpu_total": 8, "cpu_usage": 0.06011962890625, "memory": 1786.0, "memory_total": 11944.0, "memory_usage": 0.14953114534494308, "pipeline_run_id": 393, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 393 in schedule 5 is alive.", "timestamp": 1759235754.262118, "uuid": "46b73e6927134578bf0d06b25f542256"}
2025-09-30T12:36:04 {"cpu": 0.4921875, "cpu_total": 8, "cpu_usage": 0.0615234375, "memory": 1844.0, "memory_total": 11944.0, "memory_usage": 0.15438713998660417, "pipeline_run_id": 393, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 393 in schedule 5 is alive.", "timestamp": 1759235764.197209, "uuid": "1a6fd37b4ca6455d9312b388484c19e4"}
2025-09-30T12:36:14 {"cpu": 0.5302734375, "cpu_total": 8, "cpu_usage": 0.0662841796875, "memory": 2033.0, "memory_total": 11944.0, "memory_usage": 0.17021098459477563, "pipeline_run_id": 393, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 393 in schedule 5 is alive.", "timestamp": 1759235774.385185, "uuid": "7ad254cf8262434dba640c35265ac38e"}
2025-09-30T12:36:25 {"cpu": 0.64306640625, "cpu_total": 8, "cpu_usage": 0.08038330078125, "memory": 8278.0, "memory_total": 11944.0, "memory_usage": 0.693067649028801, "pipeline_run_id": 393, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 393 in schedule 5 is alive.", "timestamp": 1759235785.083912, "uuid": "b255368507c0425aa130fd8c2d4245b7"}
2025-09-30T12:36:33 {"cpu": 0.6845703125, "cpu_total": 8, "cpu_usage": 0.0855712890625, "memory": 8931.0, "memory_total": 11944.0, "memory_usage": 0.7477394507702613, "pipeline_run_id": 393, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 393 in schedule 5 is alive.", "timestamp": 1759235793.812858, "uuid": "ca0d89da4506464ea4aa231e53ee78bf"}
2025-09-30T12:36:44 {"cpu": 0.71533203125, "cpu_total": 8, "cpu_usage": 0.08941650390625, "memory": 10064.0, "memory_total": 11944.0, "memory_usage": 0.8425987943737442, "pipeline_run_id": 393, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 393 in schedule 5 is alive.", "timestamp": 1759235804.710201, "uuid": "42964f91e054431280e67e47388d1e74"}
2025-09-30T12:36:54 {"cpu": 0.76708984375, "cpu_total": 8, "cpu_usage": 0.09588623046875, "memory": 10050.0, "memory_total": 11944.0, "memory_usage": 0.8414266577361018, "pipeline_run_id": 393, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 393 in schedule 5 is alive.", "timestamp": 1759235814.208063, "uuid": "b2e1af422ae04d87a76aca279759df01"}
2025-09-30T12:37:04 {"cpu": 0.8076171875, "cpu_total": 8, "cpu_usage": 0.1009521484375, "memory": 10418.0, "memory_total": 11944.0, "memory_usage": 0.8722371064969859, "pipeline_run_id": 393, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 393 in schedule 5 is alive.", "timestamp": 1759235824.042967, "uuid": "1c3ffc9612794f8b95ab55f0a6f9f36c"}
2025-09-30T12:37:15 {"cpu": 0.94970703125, "cpu_total": 8, "cpu_usage": 0.11871337890625, "memory": 10127.0, "memory_total": 11944.0, "memory_usage": 0.8478734092431346, "pipeline_run_id": 393, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 393 in schedule 5 is alive.", "timestamp": 1759235835.004505, "uuid": "f95dcce8900c4f5e8468e71b37451a3d"}
2025-09-30T12:37:15 {"pipeline_run_id": 393, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "block_run_id": 1124, "block_uuid": "ingest_taxi_trips", "hostname": "3d7c559945b8", "level": "INFO", "message": "Process 56 is dead for job block_run_1124", "timestamp": 1759235835.348105, "uuid": "d38f0a801888478dba303f1b80ee430e"}
2025-09-30T12:37:24 {"cpu": 1.4228515625, "cpu_total": 8, "cpu_usage": 0.1778564453125, "memory": 9460.0, "memory_total": 11944.0, "memory_usage": 0.7920294708640322, "pipeline_run_id": 393, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 393 in schedule 5 is alive.", "timestamp": 1759235844.674732, "uuid": "874f9563ba114cba9aa165d8956e16bb"}
2025-09-30T12:37:26 {"block_run_id": 1124, "block_uuid": "ingest_taxi_trips", "pipeline_run_id": 393, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Execute PipelineRun 393, BlockRun 1124: pipeline ny_taxi_pipeline block ingest_taxi_trips", "timestamp": 1759235846.141946, "uuid": "0d1cae40312d41e382ab5de7e36794bb"}
2025-09-30T12:37:30 {"block_run_id": 1124, "block_uuid": "ingest_taxi_trips", "pipeline_run_id": 393, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "EXCEPTION", "message": "BlockRun 1124 (block_uuid: ingest_taxi_trips) failed.", "timestamp": 1759235850.019084, "uuid": "db960ddae9a846b5be00af07cee0372c", "error": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py\", line 647, in execute\n    result = __execute_with_retry()\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/shared/retry.py\", line 54, in retry_func\n    raise e\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/shared/retry.py\", line 38, in retry_func\n    return func(*args, **kwargs)\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py\", line 621, in __execute_with_retry\n    return self._execute(\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py\", line 1154, in _execute\n    result = self.block.execute_sync(\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py\", line 1691, in execute_sync\n    return __execute()\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py\", line 1668, in __execute\n    raise err\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py\", line 1570, in __execute\n    output = self.execute_block(\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py\", line 1910, in execute_block\n    outputs = self._execute_block(\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py\", line 2073, in _execute_block\n    outputs = self.execute_block_function(\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py\", line 2169, in execute_block_function\n    output = block_function_updated(*input_vars, **global_vars)\n  File \"<string>\", line 18, in load_data_from_file\n  File \"/usr/local/lib/python3.10/site-packages/pandas/io/parquet.py\", line 503, in read_parquet\n    return impl.read(\n  File \"/usr/local/lib/python3.10/site-packages/pandas/io/parquet.py\", line 244, in read\n    path_or_handle, handles, kwargs[\"filesystem\"] = _get_path_or_handle(\n  File \"/usr/local/lib/python3.10/site-packages/pandas/io/parquet.py\", line 102, in _get_path_or_handle\n    handles = get_handle(\n  File \"/usr/local/lib/python3.10/site-packages/pandas/io/common.py\", line 713, in get_handle\n    ioargs = _get_filepath_or_buffer(\n  File \"/usr/local/lib/python3.10/site-packages/pandas/io/common.py\", line 363, in _get_filepath_or_buffer\n    with urlopen(req_info) as req:\n  File \"/usr/local/lib/python3.10/site-packages/pandas/io/common.py\", line 265, in urlopen\n    return urllib.request.urlopen(*args, **kwargs)\n  File \"/usr/local/lib/python3.10/urllib/request.py\", line 216, in urlopen\n    return opener.open(url, data, timeout)\n  File \"/usr/local/lib/python3.10/urllib/request.py\", line 525, in open\n    response = meth(req, response)\n  File \"/usr/local/lib/python3.10/urllib/request.py\", line 634, in http_response\n    response = self.parent.error(\n  File \"/usr/local/lib/python3.10/urllib/request.py\", line 563, in error\n    return self._call_chain(*args)\n  File \"/usr/local/lib/python3.10/urllib/request.py\", line 496, in _call_chain\n    result = func(*args)\n  File \"/usr/local/lib/python3.10/urllib/request.py\", line 643, in http_error_default\n    raise HTTPError(req.full_url, code, msg, hdrs, fp)\nurllib.error.HTTPError: HTTP Error 403: Forbidden\n", "error_stack": ["  File \"/usr/local/bin/mage\", line 8, in <module>\n    sys.exit(app())\n", "  File \"/usr/local/lib/python3.10/site-packages/typer/main.py\", line 311, in __call__\n    return get_command(self)(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/click/core.py\", line 1442, in __call__\n    return self.main(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/typer/core.py\", line 778, in main\n    return _main(\n", "  File \"/usr/local/lib/python3.10/site-packages/typer/core.py\", line 216, in _main\n    rv = self.invoke(ctx)\n", "  File \"/usr/local/lib/python3.10/site-packages/click/core.py\", line 1830, in invoke\n    return _process_result(sub_ctx.command.invoke(sub_ctx))\n", "  File \"/usr/local/lib/python3.10/site-packages/click/core.py\", line 1226, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n", "  File \"/usr/local/lib/python3.10/site-packages/click/core.py\", line 794, in invoke\n    return callback(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/typer/main.py\", line 683, in wrapper\n    return callback(**use_params)  # type: ignore\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/cli/main.py\", line 147, in start\n    start_server(\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/server/server.py\", line 809, in start_server\n    scheduler_manager.start_scheduler()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/server/scheduler_manager.py\", line 108, in start_scheduler\n    proc.start()\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 121, in start\n    self._popen = self._Popen(self)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 224, in _Popen\n    return _default_context.get_context().Process._Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 281, in _Popen\n    return Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 19, in __init__\n    self._launch(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 71, in _launch\n    code = process_obj._bootstrap(parent_sentinel=child_r)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 314, in _bootstrap\n    self.run()\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 108, in run\n    self._target(*self._args, **self._kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/db/process.py\", line 15, in start_session_and_run\n    results = target(*args)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/server/scheduler_manager.py\", line 60, in run_scheduler\n    LoopTimeTrigger().start()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/triggers/loop_time_trigger.py\", line 14, in start\n    self.run()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/triggers/time_trigger.py\", line 11, in run\n    schedule_all()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/pipeline_scheduler_original.py\", line 1819, in schedule_all\n    PipelineScheduler(r).schedule()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/db/__init__.py\", line 159, in func_with_rollback\n    return func(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/pipeline_scheduler_original.py\", line 334, in schedule\n    self.__schedule_blocks(block_runs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/pipeline_scheduler_original.py\", line 641, in __schedule_blocks\n    job_manager.add_job(\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/job_manager.py\", line 21, in add_job\n    self.queue.enqueue(job_id, target, *args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/queue/process_queue.py\", line 122, in enqueue\n    self.start_worker_pool()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/queue/process_queue.py\", line 199, in start_worker_pool\n    self.worker_pool_proc.start()\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 121, in start\n    self._popen = self._Popen(self)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 224, in _Popen\n    return _default_context.get_context().Process._Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 281, in _Popen\n    return Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 19, in __init__\n    self._launch(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 71, in _launch\n    code = process_obj._bootstrap(parent_sentinel=child_r)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 314, in _bootstrap\n    self.run()\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 108, in run\n    self._target(*self._args, **self._kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/queue/process_queue.py\", line 353, in poll_job_and_execute\n    worker.start()\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 121, in start\n    self._popen = self._Popen(self)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 224, in _Popen\n    return _default_context.get_context().Process._Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 281, in _Popen\n    return Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 19, in __init__\n    self._launch(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 71, in _launch\n    code = process_obj._bootstrap(parent_sentinel=child_r)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 314, in _bootstrap\n    self.run()\n", "  File \"/usr/local/lib/python3.10/site-packages/newrelic/api/background_task.py\", line 117, in wrapper\n    return wrapped(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/queue/process_queue.py\", line 317, in run\n    start_session_and_run(args[1], *args[2], **args[3])\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/db/process.py\", line 15, in start_session_and_run\n    results = target(*args)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/pipeline_scheduler_original.py\", line 1311, in run_block\n    return ExecutorFactory.get_block_executor(\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py\", line 682, in execute\n    on_failure(\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/db/__init__.py\", line 159, in func_with_rollback\n    return func(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/pipeline_scheduler_original.py\", line 465, in on_block_failure\n    self.logger.exception(\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/logging/logger.py\", line 30, in exception\n    self.__send_message('exception', message, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/logging/logger.py\", line 65, in __send_message\n    data['error_stack'] = traceback.format_stack()\n"], "error_stacktrace": "HTTP Error 403: Forbidden"}
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py", line 647, in execute
    result = __execute_with_retry()
  File "/usr/local/lib/python3.10/site-packages/mage_ai/shared/retry.py", line 54, in retry_func
    raise e
  File "/usr/local/lib/python3.10/site-packages/mage_ai/shared/retry.py", line 38, in retry_func
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py", line 621, in __execute_with_retry
    return self._execute(
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py", line 1154, in _execute
    result = self.block.execute_sync(
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py", line 1691, in execute_sync
    return __execute()
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py", line 1668, in __execute
    raise err
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py", line 1570, in __execute
    output = self.execute_block(
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py", line 1910, in execute_block
    outputs = self._execute_block(
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py", line 2073, in _execute_block
    outputs = self.execute_block_function(
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py", line 2169, in execute_block_function
    output = block_function_updated(*input_vars, **global_vars)
  File "<string>", line 18, in load_data_from_file
  File "/usr/local/lib/python3.10/site-packages/pandas/io/parquet.py", line 503, in read_parquet
    return impl.read(
  File "/usr/local/lib/python3.10/site-packages/pandas/io/parquet.py", line 244, in read
    path_or_handle, handles, kwargs["filesystem"] = _get_path_or_handle(
  File "/usr/local/lib/python3.10/site-packages/pandas/io/parquet.py", line 102, in _get_path_or_handle
    handles = get_handle(
  File "/usr/local/lib/python3.10/site-packages/pandas/io/common.py", line 713, in get_handle
    ioargs = _get_filepath_or_buffer(
  File "/usr/local/lib/python3.10/site-packages/pandas/io/common.py", line 363, in _get_filepath_or_buffer
    with urlopen(req_info) as req:
  File "/usr/local/lib/python3.10/site-packages/pandas/io/common.py", line 265, in urlopen
    return urllib.request.urlopen(*args, **kwargs)
  File "/usr/local/lib/python3.10/urllib/request.py", line 216, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/local/lib/python3.10/urllib/request.py", line 525, in open
    response = meth(req, response)
  File "/usr/local/lib/python3.10/urllib/request.py", line 634, in http_response
    response = self.parent.error(
  File "/usr/local/lib/python3.10/urllib/request.py", line 563, in error
    return self._call_chain(*args)
  File "/usr/local/lib/python3.10/urllib/request.py", line 496, in _call_chain
    result = func(*args)
  File "/usr/local/lib/python3.10/urllib/request.py", line 643, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 403: Forbidden
2025-09-30T12:37:34 {"cpu": 1.4619140625, "cpu_total": 8, "cpu_usage": 0.1827392578125, "memory": 9586.0, "memory_total": 11944.0, "memory_usage": 0.8025787006028131, "pipeline_run_id": 393, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 393 in schedule 5 is alive.", "timestamp": 1759235854.337256, "uuid": "f6e477172e594fcea1541a41797aa01a"}
